src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/grid.controller.js

```diff
@@ -343,6 +343,37 @@ angular.module("umbraco")
             }
         };
 
+        var shouldApply = function(item, itemType, gridItem) {
+            if (item.applyTo === undefined || item.applyTo === null || item.applyTo === "") {
+                return true;
+            }
+
+            if (typeof (item.applyTo) === "string") {
+                return item.applyTo === itemType;
+            }
+
+            if (itemType === "row") {
+                if (item.applyTo.row === undefined) {
+                    return false;
+                }
+                if (item.applyTo.row === null || item.applyTo.row === "") {
+                    return true;
+                }
+                var rows = item.applyTo.row.split(',');
+                return _.indexOf(rows, gridItem.name) !== -1;
```

Comment:
Is this underscore helper apply to the array of split values in the rows variable @clausjensen or is this simply finding only the first row that matches the current grid item row name?